### PR TITLE
tests: fix some flakiness with e2e tests

### PR DESCRIPTION
**Pull request by leafty**

### CODE REVIEW

**Review by ciyer — APPROVED** [review]
This fixes the problems that I was seeing locally!

File: client/src/project/Project.present.js [hunk 1]
```diff
@@ -800,55 +798,55 @@ class ProjectViewFiles extends Component {
...
+//             </Route>
+//             <Route exact path={props.settingsCloudStorageUrl}>
+//               <ProjectSettingsCloudStorage />
+//             </Route>
+//           </Switch>
+//         </Col>
+//       </Row>
+//     </Col>
+//   );
+// }
```
Comment (lorenzo-cavazzi) [inline client/src/project/Project.present.js:849]:
```suggestion
```
Should we remove this before merging?

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
I dislike adding a re-try strategy for local tests, but it seems effective and gives us time to improve the UI wherever needed.

image

File: client/src/project/Project.present.js (hunk 1 above)
Comment (leafty) [reply client/src/project/Project.present.js]: Done in [fc2b93c](https://github.com/SwissDataScienceCenter/renku-ui/pull/2872/commits/fc2b93c4611d14f78b7f2e2a1163b05b4f447cfd).

**Review by lorenzo-cavazzi — APPROVED** [review]
Thank you for the joint effort on this :pray:

**Pull request merged**